cmd/pulumi-test-language/l2resourcesimple_test.go

```diff
@@ -356,5 +360,5 @@ func TestL2SimpleResource_MissingRequiredPlugins(t *testing.T) {
 	failureMessage := runResponse.Messages[0]
 	assert.Contains(t, failureMessage,
 		"expected no error, got Error: unexpected required plugins: "+
-			"actual [language-mock@<nil>], expected [language-mock@<nil> resource-simple@1.0.0]")
+			"actual Set{language-mock@<nil>}, expected Set{language-mock@<nil>, resource-simple@1.0.0}")
```

Comment:
Oh didn't this change a while ago? Are we not running these tests in CI? I'll see if I can raise a quick PR to fix that.